autostart/qubes-app-menu.desktop

```diff
@@ -1,9 +1,9 @@
 [Desktop Entry]
 Type=Application
-Exec=systemctl --user start qubes-app-menu
+Exec=sh -c "systemctl --user import-environment XDG_CURRENT_DESKTOP; systemctl --user start qubes-app-menu"
 Icon=qubes-manager
 Terminal=false
 Name=Qubes Application Menu
 GenericName=Qubes Application Menu
 StartupNotify=false
-Hidden=True
+Hidden=False
```

Comment:
I'd also consider `NotShowIn=X-QUBES`, to not start it automatically in AppVM (if you have a template prepared for GUI VM).